tsconfig.cjs.json

```diff
@@ -1,10 +1,13 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": {
-    "importHelpers": true,
+    "lib": ["es2019", "es2020.promise", "es2020.bigint", "es2020.string"],
     "module": "commonjs",
-    "noEmitHelpers": true,
-    "target": "ES2018",
+    "target": "es2019",
+
```

Comment:
extra line?

Reply:
I wanted to demarcate main key: i.e. lib, module, target from the rest.
We should bump to yarn berry, and put constraints for such and other rules.

Anyway, we can't merge this change as CI is failing, as we define tsconfig.cjs.json for a browser package.
We should write a yarn berry constraints for browser, node, and universal packages and a standardized way for defining runtime specific packages.